docs/docs/community/create-provider.md

```diff
@@ -1,60 +1,76 @@
 ---
 sidebar_position: 4
+sidebar_label: 'Add a new provider'
 ---
 
-# How to add a Provider?
+# How to add a new provider?
 
-## Creating a Novu provider package
+Novu currently supports five channels `in_app`, `push`, `email`, `chat` and `sms`. For `in_app` we support only our own provider, so new provider cannot be added for this channel. For other four channels we support integration of external providers. This guide will help in adding new provider for any of these 4 channels. **In this guide, we are adding a new provider for email channel, but all of the mentioned steps are similar for other channels as well**.
 
-All of our providers need to implement one or more of our provider interfaces, based on provider feature, from Email, and SMS through Chat, In-app, and push.
+## Description
 
-For a provider template, you can copy one of our existing provider from the `providers` folder in the Novu project, make the relevant changes and create a PR against the monorepo.
+Providers allow us to handle message delivery over multiple channels. We have multiple providers for each channel (SMS, Email, Push and Chat). To get started with adding a new provider let's look at setting up our repository.
 
-### Description
+## Initialization
 
-Providers allow us to handle message delivery over multiple channels. We have multiple providers for each channel (SMS, Email, Push, Chat, and others). To get started with adding a new provider let's look at setting up our repository.
+Fork the novu repository and clone it in your local machine.
 
-### Initialization
+```bash
+git clone https://github.com/<'YOUR_GITHUB_USER_NAME'>/novu.git
+```
 
 To set up the repository, run the initial setup command:
 
-```zsh
+```bash
 npm run setup:project
 ```
 
 At the root of the project build the `node` package to get started.
 
-```zsh
+```bash
 cd packages/node && npm run build
 ```
 
-### Snippets
+## Generate provider
 
-After the project is initialized creating a new provider is achievable with the following.
+After the project is initialized, new provider can be generated using below command.
 
-```zsh
-yarn run generate:provider
+```bash
+npm run generate:provider
```

Comment:
```suggestion
pnpm run generate:provider
```
We use `pnpm` in our monorepo.

Reply:
Shall I replace npm with pnpm at all commands in this page?

Reply:
I think so for coherence. Running the monorepo needs `pnpm`.